api/v1/ovsnetwork_types.go

```diff
@@ -0,0 +1,91 @@
+/*
+Copyright 2021.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package v1
+
+import (
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+)
+
+// OVSNetworkSpec defines the desired state of OVSNetwork
+type OVSNetworkSpec struct {
+	// Namespace of the NetworkAttachmentDefinition custom resource
+	NetworkNamespace string `json:"networkNamespace,omitempty"`
+	// OVS Network device plugin endpoint resource name
+	ResourceName string `json:"resourceName"`
+	// Capabilities to be configured for this network.
+	// Capabilities supported: (mac|ips), e.g. '{"mac": true}'
+	Capabilities string `json:"capabilities,omitempty"`
```

Comment:
can we use `map[string]string` here?

Reply:
Theoretically we can use `map[string]bool` here, but then we will have inconsistency in the  API, SriovNetwork and SriovIBNetwork are using json as a payload for capabilities field. I would prefer to keep all APIs aligned.